app/src/Menus/Menu.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import * as apiClient from "../apiClient";
+import WhiteBorder from "../images/border-style-white.png";
+
+import CourseType from "./CourseType";
+import FilterSection from "./FilterSection";
+import styles from "./styles.module.scss";
+
+function Menu({ type }) {
+  const [info, setInfo] = React.useState([]);
+  const [fullMenu, setFullMenu] = React.useState({});
+  const courses = info[`${type.toLowerCase()}_courses`];
+
+  //store food items
+  const [foodItems, setFoodItems] = React.useState([]);
+  //store checked filter boxes
+
+  const loadInfo = async () => {
+    setInfo(await apiClient.getInfo());
+  };
+
+  const loadFoodItems = async () => {
+    setFoodItems(await apiClient.getFoodItems(type));
+    setFullMenu(await apiClient.getFoodItems(type));
```

Comment:
Can you call API client once?

```suggestion
    const foodItems = await apiClient.getFoodItems(type);
    
    setFoodItems(foodItems);
    setFullMenu(foodItems);
```